pkg/api/types.go

```diff
@@ -438,6 +440,21 @@ type GlusterfsVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// CinderPersistentDisk represents a Persistent Disk resource in Openstack.
+// A Cinder volume must exist and be formatted before mounting to a container.
+// The disk must also be in the same region as the kubelet.
+type CinderPersistentDiskVolumeSource struct {
```

Comment:
Yeah "PersistentDisk" is not a general concept name, it's literally the product name on Google Cloud Platform.  If it's not part of the well-known name of this concept, don't include the words.